axelar/assetlist.json

```diff
@@ -1380,6 +1366,50 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/ox.svg"
       }
+    },
+    {
+      "description": "The Axelar Bridged token of Cacao Swap",
+      "denom_units": [
+        {
+          "denom": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+          "exponent": 0
+        },
+        {
+          "denom": "yum.axl",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+      "name": "Axelar Wrapped YUM",
+      "display": "yum.axl",
+      "symbol": "YUM.axl",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "arbitrum",
```

Comment:
is there an IBC connection between axelar and arbitrum? I didn't know arbitrum had IBC... If this exists, it would need to be registered under /_IBC/

Reply:
Hey @JeremyParish69 sorry about this (first time doing it). So, our token is on Arbitrum, but also we bridged it with axelar. So we actualkly have YUM (on arb) , and YUM.axl (on axelar). 

Any guidance on which one to use?